morepath/core.py

```diff
@@ -81,7 +80,7 @@ def request_method_predicate(request):
     return request.method
 
 
-@App.predicate_fallback(generic.view, request_method_predicate)
+@App.predicate_fallback(App.get_view, request_method_predicate)
```

Comment:
The fallback declarations on the dispatchmethods-decoupled branch get *three* arguments: ``self`` (i.e. app), obj and request. This so to avoid a wrapper function -- need to read directive.py to see what you do.